README.md

```diff
@@ -2656,46 +2253,8 @@ Other Style Guides
     }
     ```
 
-  <a name="whitespace--in-parens"></a><a name="18.9"></a>
```

Comment:
For whitespace in parens and brackets we should have a rule if we should do it not. I for one like airbnb's rule, but don't have a strong opinion about it.

Only thing i want is to only have one way of doing it, not 2.